examples/hydrogen-ibmq.yaml

```diff
@@ -0,0 +1,167 @@
+ZapOSApiVersion: v1alpha1
+kind: Workflow
+
+resources:
+- name: diatomic-molecule
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/tutorial-1-diatomic-molecule.git"
+    branch: "dev"
+- name: z-quantum-core
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/z-quantum-core.git"
+    branch: "dev"
+- name: qe-openfermion
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/qe-openfermion.git"
+    branch: "dev"
+- name: qe-psi4
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/qe-psi4.git"
+    branch: "dev"
+- name: z-quantum-vqe
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/z-quantum-vqe.git"
+    branch: "dev"
+- name: z-quantum-optimizers
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/z-quantum-optimizers.git"
+    branch: "dev"
+- name: qe-forest
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/qe-forest.git"
+    branch: "dev"
+- name: qe-qiskit
+  type: git
+  parameters:
+    url: "git@github.com:zapatacomputing/qe-qiskit.git"
+    branch: "ibm-vqf"
```

Comment:
branch should now point to `"master"` or `"dev"`

Reply:
Changed to `dev`